common/model/tryte_transaction.cc

```diff
@@ -50,7 +50,7 @@ std::string TryteTransaction::message(void) {
   return trytes;
 }
 
-// Set the transaction message (copy argument)
+// Set the transaction message (assign)
 void TryteTransaction::setMessage(const std::string &trytes) {
   size_t flex_len = flex_trits_num_for_trits(NUM_TRITS_SIGNATURE);
   flex_trit_t trits[flex_len];
```

Comment:
All that the `transaction_set_message` methods do is memcpy.
Can't you just memset & copy straight into the transaction's structure? 
All of these methods waste memory